metabasic/metabasic.py

```diff
@@ -5,6 +5,8 @@
 
 from .exceptions import AuthError, ConfigError
 
+import pandas as pd
```

Comment:
this should be grouped with the external dependency inquirer/requests imports re: [pep8](https://www.python.org/dev/peps/pep-0008/#id23) 

isort should handle this for you, code formatting can be run via the `make format` target